tests/unittest/_torch/modeling/test_modeling_multimodal.py

```
@@ -370,10 +370,12 @@ def get_trtllm_inputs(
             position_ids = [torch.arange(0, input_ids.size(-1), dtype=torch.int32)]
             multimodal_runtime = (
                 MultimodalRuntimeData(
-                    mm_token_lengths=multimodal_params_list[0].multimodal_input.multimodal_lengths,
-                    mm_token_positions=multimodal_params_list[
-                        0
-                    ].multimodal_input.multimodal_positions,
+                    mm_contiguous_spans=list(
+                        zip(
+                            multimodal_params_list[0].multimodal_input.multimodal_positions,
+                            multimodal_params_list[0].multimodal_input.multimodal_lengths,
+                        )
```

Ruff (0.15.10)

[warning] `zip()` without an explicit `strict=` parameter

Add explicit value for parameter `strict=`

(B905)